queue/src/main/java/org/killbill/queue/DBBackedQueue.java

```diff
@@ -282,39 +282,28 @@ public Void inTransaction(final QueueSqlDao<T> transactional, final TransactionS
                     return null;
                 }
 
-                final Collection<T> entriesToMove = new ArrayList<T>(entriesLeftBehind.size());
                 final List<T> entriesToReInsert = new ArrayList<T>(entriesLeftBehind.size());
-                final List<T> stuckEntries = new LinkedList<T>();
                 final List<T> lateEntries = new LinkedList<T>();
                 for (final T entryLeftBehind : entriesLeftBehind) {
                     // entryIsBeingProcessedByThisNode is a sign of a stuck entry on this node
                     final boolean entryIsBeingProcessedByThisNode = owner.equals(entryLeftBehind.getProcessingOwner());
```

Comment:
Not used anymore.

Reply:
Fixed in [158f874](https://github.com/killbill/killbill-commons/pull/170/commits/158f8747ef578aa4b1fa35da8fe1f2262f0084fa)